components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.html

```diff
@@ -14,15 +14,34 @@
   </button>
   <chef-click-outside (clickOutside)="closeDropdown()">
     <chef-dropdown [attr.visible]="active">
-      <chef-checkbox
-        *ngFor="let project of projectsArray"
-        [checked]="project.checked"
-        [attr.title]="project.name"
-        (change)="projectChecked($event.detail, project)"
-        (keydown.enter)="closeDropdown()"
-        (keydown.esc)="closeDropdown()"
-        (keydown.arrowup)="moveFocus($event)"
-        (keydown.arrowdown)="moveFocus($event)">{{ project['name'] }}</chef-checkbox>
+
+      <div id="filter-container">
+        <input 
```

Comment:
is there a plan to make a `filter` component?

Reply:
I would like to - I think it would be nice to have, especially if we add a few more of these